apps/studio/src/mixins/data_converter.js

```diff
@@ -13,9 +13,14 @@ export default {
               ];
             */
           const columnNamesOnly = columns.map((c) => c.field)
-          return data.rows.map((row) => {
-          return _.pick(row, columnNamesOnly)
-          })
+          return data.rows.map((row) => _
+            .chain(row)
+            .pick(columnNamesOnly)
+            .transform((result, val, key) => {
+              result[key] = Array.isArray(val) ? `{${val.join(',')}}` : val
```

Comment:
Do you need to escape any  characters in the string? Maybe no, but worth a check.